src/ios-plugin/swift-lang/src/main/java/io/ecocode/swift/antlr/generated/Swift5ParserBaseListener.java

```diff
@@ -0,0 +1,3884 @@
+/*
```

Comment:
generated files would not be part fo source code but in target directory to follow dev best practices

Reply:
It is not actually these kind of files generated on each build...
It's the ANTLR AST parser: it is generated only once. And will be generated only when we want to upgrade Swift language version support.
See it more like "scaffolded code". I put it in a "generated" package just to denote that developers do not have control onto it and should never change it manually.
If you have a more suitable name for the package, I can change it.

Reply:
you are the expert of IOS and not me :p 
but to understand this purpose, I think it will be documented on `CONTRIBUTING.md` file for example.

Reply:
It's not iOS related: it's the way ANTLR works :)
Anyway: I wrote a not about it into CONTRIBUTING.md